src/skydb.test.ts

```diff
@@ -20,6 +20,15 @@ describe("User", () => {
       expect(new User(username, password).id).toEqual(expected.id);
     }
   });
+
+  it.skip("should take a considerable amount of time", async () => {
+    const start = new Date().getTime();
+    new User("john.doe@example.com", "supersecret");
+    const elapsed = new Date().getTime() - start;
+    expect(elapsed).toBeGreaterThan(10);
+    expect(elapsed).toBeLessThan(100);
+    console.log(`${elapsed}ms`);
+  });
```

Comment:
what does this test do?

Reply:
quickly wrote that to verify how long it took to create a new user on my machine, 
was only looking for order of magnitudes here while playing with the key derivation function input
(like `numIterations` and digest function)

Reply:
either remove that or make it a proper test before merging to master please

Reply:
Not sure what you mean with "proper test". I assume document more? Not sure though, it's hardware dependant and I don't think it's worth the effort. That is also why I originally ended up skipping it and now decided to remove it all together.